src/plugins/link/test/ts/browser/ForceExternalTargetsTest.ts

```diff
@@ -0,0 +1,47 @@
+import '../../../../../themes/silver/main/ts/Theme';
+
+import { Pipeline, Log } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import LinkPlugin from 'tinymce/plugins/link/Plugin';
+
+import { TestLinkUi } from '../module/TestLinkUi';
+
+UnitTest.asynctest('browser.tinymce.plugins.link.AssumeExternalTargetsTest', (success, failure) => {
+
+  LinkPlugin();
+
+  TinyLoader.setup(function (editor, onSuccess, onFailure) {
+    const tinyApis = TinyApis(editor);
+
+    Pipeline.async({},
+      Log.steps('TBA', 'Link: Test that with default setting, URLs are not modified', [
```

Comment:
If you use `Log.stepsAsStep` you can then pass an array of these to `Pipeline.async`, which will provide better logging of the separate things your test is doing

Reply:
As a side note, nice work on getting this test written! :+1:

Reply:
Thanks, updated in https://github.com/tinymce/tinymce/pull/5020.